google/cloud/bigquery/client.py

```diff
@@ -526,12 +538,12 @@ def _ensure_bqstorage_client(
             bqstorage_client = bigquery_storage.BigQueryReadClient(
                 credentials=self._credentials,
                 client_options=client_options,
-                client_info=client_info,
+                client_info=client_info,  # type: ignore  # (None is also accepted)
```

Comment:
It appears so, yes. The [docstring](https://github.com/googleapis/python-bigquery-storage/blob/acc92249013f1b31fdac2aa4bf5a6864730d7422/google/cloud/bigquery_storage_v1/services/big_query_read/client.py#L305-L309) says that `None` is allowed, but `mypy` complains that `Optional[...ClientInfo]` is not acceptable here as an argument.

It's probably because the default value of `client_info` is not a literal `None`, but `DEFAULT_CLIENT_INFO` instead.